docs/kind.md

```diff
@@ -84,7 +84,6 @@ configuration options when deploying. Use `./kind.sh -h` to see the latest optio
 usage: kind.sh [[[-cf |--config-file <file>] [-kt|keep-taint] [-ha|--ha-enabled]
                  [-ho |--hybrid-enabled] [-ii|--install-ingress] [-n4|--no-ipv4]
                  [-i6 |--ipv6] [-wk|--num-workers <num>] [-ds|--disable-snat-multiple-gws]
-                 [-dp |--disable-pkt-mtu-check]
```

Comment:
Should this be part of the previous "Handling packet sizes greater than pod MTU" patch?

Reply:
This change was introduced in a separate commit (9137c9a31196373e2fdadaa2917383d8b3f51828) which aims to improve kind documentation, not mtu specific commit.